cube/helm/cube/values.yaml.tpl

```diff
@@ -13,12 +13,20 @@ cube:
   config:
     apiSecret: {{ $apiSecret }}
 
-{{ if $isGcp }}
+{{ if or ($isGcp) (eq .Provider "aws) }}
 cubestore:
   cloudStorage:
+    {{ if $isGcp }}
     gcp:
       credentialsFromSecret:
         name: cube-gcp-credentials
         key: key.json
       bucket: {{ .Values.cubeBucket }}
+    {{ else if eq .Provider "aws" }}
+    aws:
+      accessKeyID: {{ importValue "Terraform" "iam_access_key_id" }}
```

Comment:
it would be better to use IRSA instead of fixed keys, I'm not sure if cubejs supports it but might be worth looking

Reply:
Unfortunately, cubejs doesn't seems to support IRSA (nor workload identity)